bottlecap/src/logs/datadog.rs

```diff
@@ -0,0 +1,48 @@
+use std::{env, error::Error};
+
+use tracing::{debug, error};
+
+use crate::logs::processor::IntakeLog;
+
+const LOGS_API_INTAKE: &str = "https://http-intake.logs.datadoghq.com/api/v2/logs";
+
+pub struct DdApi {
+    api_key: String,
+    ureq_agent: ureq::Agent,
+}
+
+impl DdApi {
+    pub fn new() -> Self {
+        DdApi {
+            api_key: env::var("DD_API_KEY").expect("unable to read envvars, catastrophic failure"),
```

Comment:
the API key could live in datadog.yml file as well, maybe it would be easier to get it from the config?

Reply:
Yup, I'm going to create another PR for this